games-roguelike/dwarf-fortress/dwarf-fortress-0.47.05-r1.ebuild

```diff
@@ -60,7 +61,11 @@ src_compile() {
 	# -DDEBUG is recognized to give additional debug output
 	append-cppflags -D$(usev !debug N)DEBUG
 
-	emake -f "${FILESDIR}"/Makefile.native
+	if use gui; then
+		emake -f "${FILESDIR}"/Makefile.native
+	else
+		emake -f "${FILESDIR}"/Makefile.native HAVE_GTK=0
+	fi
```

Comment:
This could be:

```
emake -f "${FILESDIR}"/Makefile.native HAVE_GTK=$(usex gui 1 0)
```

Reply:
Upstream is a single developer who does not disclose the source code and does not accept changes from the community (unless in the form of mods), so we don't have a choice.

I agree concerning the $HOME problem. I have a few modifications to my own build to stick it in a different dir but this wouldn't be everybody's cup of tea.

I'll make your suggested change soon and push.